docs/proto-docs.md

```diff
@@ -5372,6 +5374,23 @@ RemoveMsgFeeProposal defines a governance proposal to delete a current msg based
 
 
 
+<a name="provenance.msgfees.v1.UpdateConversionFeeDenomProposal"></a>
+
+### UpdateConversionFeeDenomProposal
+UpdateConversionFeeDenomProposal defines a governance proposal to update the msg fee conversion denom
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `title` | [string](#string) |  | proposal title |
+| `description` | [string](#string) |  | proposal description |
+| `conversion_fee_denom` | [string](#string) |  | conversion_fee_denom is the denom that usd will be converted to |
```

Comment:
for e.g in the usdf network this will be the ibc denom for hash on that network.